### PR TITLE
Baseline map updates

### LINKED ISSUE #620: Household data map view card snags
https://develop--micronutrientsupport-tool.netlify.app/quick-maps/diet/baseline?country-id=MWI&mnd-id=Ca&measure=diet&age-gender-group-id=WRA (opened in firefox)

- [x]  Colour ramp and legend need to reflect the loaded data - suggest try quartiles or quintiles (i.e. four or five categories) as giving a sensible number relative to the subdivisions being shown.

- [x] Is the map gonig to be exportable, as for other figures? (this may be another issue?)

- [x]  A more informative title would be good so that when the map is exported and can be taken from the data (e.g. Median apparent intake of calcium by households at District level, Malawi). 

- [x]  Map needs units displayed, which should be being calculated as Adult Female Equivalent (AFE) for the household data pages, for example mg/AFE/day

Example of suggested presentation for hover box:
image (data fields should be present in API response but please shout if unclear)

### LINKED ISSUE #692: Country level data map view card snags
different popup for household/country level day

```
 Country level:
Region Ethiopia
Dietary Availability: <value> mg per capita per day
<Below/Above> the threshold for deficiency
```

(deficientPercentage will be 0 (above) /100 (below) for country level data)

Linked issue #620

### PR DESCRIPTION
**Pull request by Fuhji**
- card title shows more information about the contents 
- mouse hover on map data shows more useful information 

close #620 @bgsandan can confirm if this MR meets the issue requirements
close #692

### CODE REVIEW

**Review by DanielLyleWilliamWarren — APPROVED** [review]
Tested and working!

**Comment by bgsandan** [comment]
Hi @Fuhji this meets the issue requirements and looks perfect for household level data **however** those data fields only really apply when the data level is household.  Is there scope to have a different popup for household/country level day

- [ ] Household as in this PR
- [ ] Country level:
    `Region Ethiopia`
    `Dietary Availability: <value> mg per capita per day`
    `<Below/Above> the threshold for deficiency`

    (deficientPercentage will be 0 (above) /100 (below) for country level data)

Does that make sense?

**Comment by Fuhji** [comment]
Sure, created #692 to add this for country level data

**Comment by Fuhji** [comment]
@bgsandan implemented to close #692 as well

File: src/app/pages/quickMaps/pages/baselineDetails/mapView/mapView.component.ts [hunk 1]
```diff
@@ -328,21 +326,31 @@ export class MapViewComponent implements AfterViewInit {
...
+        Dietary Availability (AFE): ${dietarySupplySF} ${props.unit} per day<br/>
+        Prevalence of Deficiency (EAR): ${props.deficientPercentage}% of sampled households (${props.deficientCount}/${props.householdCount})<br/>
+      </div>`;
+    } else {
+      const dietarySupplySF = this.sigFig.transform(props.dietarySupply, 6);
+      const defThreshold = props.dietarySupply < props.deficientValue ? 'Below' : 'Above';
+      return `
+      <div>
+        Region: <b>${props.areaName}</b><br/>
+        Dietary Availability (AFE): ${dietarySupplySF} ${props.unit} per capita per day<br/>
```
Comment (bgsandan) [inline src/app/pages/quickMaps/pages/baselineDetails/mapView/mapView.component.ts:350]: Can we remove `(AFE)` from this line please?  Then looks good to merge :+1:

**Review by bgsandan — APPROVED** [review]

File: src/app/pages/quickMaps/pages/baselineDetails/mapView/mapView.component.ts (hunk 1 above)
Comment (Fuhji) [reply src/app/pages/quickMaps/pages/baselineDetails/mapView/mapView.component.ts]: 👍  done

**Pull request merged**